pyproject.toml

```diff
@@ -51,6 +50,25 @@ Funding = 'https://github.com/sponsors/samuelcolvin'
 Source = 'https://github.com/pydantic/pydantic'
 Changelog = 'https://pydantic-docs.helpmanual.io/changelog'
 
+[tool.hatch.metadata.hooks.fancy-pypi-readme]
+content-type = 'text/markdown'
+fragments = [
+    { path = 'README.md' },
+    { path = 'HISTORY.md' },
```

Comment:
does this insert the new lines required?

Reply:
Yup